README.md

```diff
@@ -78,6 +78,9 @@ First, you will need to install [FFTW3](http://www.fftw.org/download.html):
 * Debian & Ubuntu: `$ sudo apt install libfftw3-dev`
 * macOS and homebrew: `$ brew install fftw`
```

Comment:
```suggestion
* Fedora: `$ sudo dnf install fftw-devel catch-devel`
* Debian & Ubuntu: `$ sudo apt install libfftw3-dev`
* macOS and homebrew: `$ brew install fftw catch2`
```

Reply:
I split it into a different paragraph because it didn't fit with the sentence above.